src/extension.ts

```diff
@@ -39,6 +39,9 @@ function runJavaServer(context: VSCode.ExtensionContext) : Thenable<StreamInfo>
 				let vmargs = getSonarLintConfiguration().get('ls.vmargs','');
 				parseVMargs(params, vmargs);
 				params.push('-jar', serverJar, '' + languageServerPort);
+				params.push(toUrl(Path.resolve(context.extensionPath, "analyzers", "sonarjs.jar")));
+				params.push(toUrl(Path.resolve(context.extensionPath, "analyzers", "sonarphp.jar")));
+				params.push(toUrl(Path.resolve(context.extensionPath, "analyzers", "sonarpython.jar")));
```

Comment:
It's not great that this file and `scripts/prepare.js` both use the hardcoded names for the jars. When updating at one place, you have to remember to update the other too, and make sure the names match precisely. Isn't there a way to have these names in a shared list, which can be updated in a single place? 

If that's not possible, then to mitigate the risk, at least it would be good to have these names in an easy-to-find place, for example in a lists defined near the top of the files, along with a comment referencing the other file that needs to be updated together.